### PR TITLE
Feat/today button

### PR DESCRIPTION
**Pull request by Inbal-Tish**

### CODE REVIEW

File: src/expandableCalendar/calendarProvider.js [hunk 1]
```diff
@@ -34,14 +42,86 @@ class CalendarProvider extends Component {
...
+  }
+
+  renderTodayButton() {
+    const todayString = XDate.locales[XDate.defaultLocale].today || commons.todayString;
+    const today = todayString.charAt(0).toUpperCase() + todayString.slice(1);
+    
+    return (
+      <Animated.View 
+        style={{
+          position: 'absolute',
```
Comment (ethanshar) [inline src/expandableCalendar/calendarProvider.js]: move this part of the style to a stylesheet

File: src/expandableCalendar/calendarProvider.js [hunk 2]
```diff
@@ -34,14 +42,86 @@ class CalendarProvider extends Component {
...
+  }
+
+  animateTodayButton(date) {
+    if (this.props.todayButton) {
+      this.setState({buttonIcon: this.getButtonIcon(date)});
+  
+      const today = XDate().toString('yyyy-MM-dd');
+      const isToday = today === date;
+      
+      Animated.spring(this.state.buttonY, {
```
Comment (ethanshar) [inline src/expandableCalendar/calendarProvider.js]: The button disappears when picking "today" day from the calendar, but if I press the actual "Today Button" it doesn't disappear.

File: src/expandableCalendar/calendarProvider.js [hunk 3]
```diff
@@ -13,15 +16,20 @@ class CalendarProvider extends Component {
     // Initial date in 'yyyy-MM-dd' format. Default = Date()
     date: PropTypes.any.isRequired,
     // callback for date change event
-    onDateChanged: PropTypes.func
+    onDateChanged: PropTypes.func,
+    // whether to show the today button
+    todayButton: PropTypes.bool
```
Comment (ethanshar) [inline src/expandableCalendar/calendarProvider.js]: maybe, name this prop `showTodayButton` so it's clear it's boolean.

Comment (Inbal-Tish) [reply src/expandableCalendar/calendarProvider.js]: Done.

File: src/expandableCalendar/calendarProvider.js (hunk 2 above)
Comment (Inbal-Tish) [reply src/expandableCalendar/calendarProvider.js]: Will check it out. Thanks!

File: src/expandableCalendar/calendarProvider.js (hunk 1 above)
Comment (Inbal-Tish) [reply src/expandableCalendar/calendarProvider.js]: Done.

File: src/expandableCalendar/agendaList.js [hunk 4]
```diff
@@ -29,11 +26,9 @@ class AgendaList extends Component {
 
   constructor(props) {
     super(props);
-
-    // this.state = {};
-
     this.style = styleConstructor(props.theme);
-    this._topSection = undefined;
+
+    this._topSection = props.sections ? props.sections[0].title : undefined;
```
[the remark this answers is not in the document]
Comment (Inbal-Tish) [reply src/expandableCalendar/agendaList.js]: Done.

File: src/expandableCalendar/calendarProvider.js (hunk 2 above)
Comment (Inbal-Tish) [reply src/expandableCalendar/calendarProvider.js]: Done. Thanks!!!

File: src/expandableCalendar/calendarProvider.js [hunk 5]
```diff
@@ -96,15 +101,17 @@ class CalendarProvider extends Component {
...
     const todayString = XDate.locales[XDate.defaultLocale].today || commons.todayString;
     const today = todayString.charAt(0).toUpperCase() + todayString.slice(1);
     
     return (
       <Animated.View style={[this.style.todayButtonContainer, {transform: [{translateY: this.state.buttonY}]}]}>
-        <TouchableOpacity style={this.style.todayButton} onPress={this.onTodayPress}>
-          <Image style={this.style.todayButtonImage} source={this.state.buttonIcon}/>
-          <Text style={this.style.todayButtonText}>{today}</Text>
-        </TouchableOpacity>
+        <Wrapper style={this.style.todayButton} onPress={this.onTodayPress}>
```
Comment (ethanshar) [inline src/expandableCalendar/calendarProvider.js]: why not just pass the `disabled` prop to `TouchableOpacity`?

**Pull request merged**